libsplinter/src/admin/rest_api/mod.rs

```diff
@@ -72,6 +72,8 @@ impl RestResourceProvider for AdminService {
 ///
 /// * `rest-api-actix`
 #[derive(Clone)]
+// Allow dead code here as this is a part of the public API but it may never be used
+#[allow(dead_code)]
```

Comment:
See the PR for 0-6 branch, which removes node_id from the struct and adds and underscore the the node_id argument on new(). That might be a better approach since it doesn't introduce any allow statements.

Reply:
0-4 branch does not have the "admin" feature which blocks pulling in all of the admin service. Because of this both the node_id and store field will come up as unused in different configurations. For example when the health service is built.